project/PACKAGERS.md

```diff
@@ -44,8 +44,7 @@ need to package Docker your way, without denaturing it in the process.
 To build Docker, you will need the following:
 
 * A recent version of Git and Mercurial
-* Go version 1.4 or later (Go version 1.5 or later required for hardware signing
-  support in Docker Content Trust)
+* Go version 1.6.2 or later
```

Comment:
So, my other question still applies: does it really need to be 1.6.2, or is just 1.6 sufficient? :smile: